### ISSUE TITLE
Latest Blast

### ISSUE THREAD

**Issue by Sivanandan** [peterjc/galaxy_blast#146]
I see that you have blast 10.1, do you plan on updating to the latest blast? We have a galaxy instance on a HPC system. On that galaxy we have blast 10.1 (from you) but we have made the latest DB available in a centralized location. Unfortunately, they aren't compatible. What do you suggest in such a situation?

**Comment by peterjc** [comment]
If you mean we currently wrap NCBI BLAST+ version 2.10.1, yes, it is a bit old.

If you are having difficulties with the DB v4 to v5 change, you can have a mixture of databases (e.g. database A in v4 format, and database B in v5 format), all in the same folder.

However, if upgrading a database from v4 to v5, you must make sure all the old files are removed. We update our NCBI databases with a script, and found with an NCBI mirrored database in v5 format, a few old v4 files still present broke things. The extensions used has changed, but the data stamps should also be informative.

**Comment by Sivanandan** [comment]
Yes, I meant 2.10.1. Thanks for your suggestion. Do you plan on wrapping the latest blast (2.13.0)?

**Comment by peterjc** [comment]
At some point, the addition of threading by query batch in BLAST+ 2.12.0 is the most note worth feature to me.

However, #140 really needs addressing first.
